README.md

```diff
@@ -102,3 +102,12 @@ straightforward using these steps:
       route `/api/kernels` is not enabled by default for greater security. See
       the `--KernelGatewayApp.list_kernels` parameter documentation if you
       would like to enable the `/api/kernels` route.)
+
+### Swagger Documentation
```

Comment:
I think these need to live in the proper docs now. The README is limited to the getting started info. I suggest moving it to https://github.com/jupyter/kernel_gateway/blob/master/docs/source/devinstall.md for now.